modules/core/include/opencv2/core/hal/intrin_rvv_scalable.hpp

```diff
@@ -543,6 +543,40 @@ OPENCV_HAL_IMPL_RVV_LOADSTORE_OP(v_float32, vfloat32m2_t, float, VTraits<v_float
 OPENCV_HAL_IMPL_RVV_LOADSTORE_OP(v_float64, vfloat64m2_t, double, VTraits<v_float64>::vlanes() / 2, VTraits<v_float64>::vlanes(), 64, f64)
 #endif
 
+template <int N = VTraits<v_uint16>::max_nlanes>
+inline void v_store(ushort* ptr, const v_uint16& a, int n = N)
```

Comment:
why `n` parameter is passed? Let's get rid of it.

Reply:
I am afraid that we cannot, since `v_store` is already defined.